admin_auto_tests/test_model.py

```diff
@@ -2,7 +2,10 @@
 
 import factory
 from django.contrib.auth import get_user_model
-from model_mommy import mommy
+try:
+    from model_bakery import baker as mommy
+except ImportError:
+    from model_mommy import mommy
```

Comment:
I highly advise you to not change the import name of model bakery to keep the previous `mommy` name working. We have these [migrations instructions](https://model-bakery.readthedocs.io/en/latest/migrating_from_mommy.html) which are very helpful to replace the `mommy` references by `baker` without breaking your test code =)